src/setupRtl.test.tsx

```diff
@@ -1,7 +1,19 @@
 import React from "react";
 
+import * as RenderRTL from "@testing-library/react";
+
 import { setupRtl } from "./setupRtl";
 
+// We mock the module and then immediately spyOn it because due to the read-only nature
+// of this module, we cannot spyOn it directly.
+// We create a mock that's the actual implementation in our mocked function as a poor-man's spyOn.
+jest.mock("@testing-library/react", () => ({
+  render: jest.fn(jest.requireActual("@testing-library/react").render),
+}));
+
+// Just a helpful way to reference our spy.
+const render = jest.spyOn(RenderRTL, "render");
```

Comment:
I think you'd have a lot easier time testing that the end component result is what you'd expect from using the `options` API. You should be able to set up tests without any `jest.mock`/`jest.spyOn` shenanigans by using the `setupRtl` function as it normally is.

Let's talk if that's not working for you?

Reply:
Oh, not a problem at all.

Just updated.

Also, to confirm, we _want_ the options to persist across multiple calls to the same `renderView` method, or is that an unintentional side effect?

I feel like it... might be messing with the segregation of each test, so figured I'd ask.

Reply:
Yeah that's something i've been wondering about personally. For now, yes, since the recommendation has been to set things up once outside of tests... but as we get more usage I'd love to reconsider.

Reply:
Right on. I'll keep it in mind as well, then!